configs/ttalps_plotter_config.py

```diff
@@ -60,14 +59,16 @@
 ]
 
 genMuonVertexCollections = [
-  "BestPFIsoDimuonVertexResonancesNotFromALP",  # Dimuon resonances
-  "GoodPFIsoDimuonVertexResonancesNotFromALP",  # Dimuon resonances
-  "BestPFIsoDimuonVertexNonresonancesNotFromALP",  # Non-resonant Dimuons
-  "GoodPFIsoDimuonVertexNonresonancesNotFromALP",  # Non-resonant Dimuons
+  # "BestPFIsoDimuonVertexResonancesNotFromALP",  # Dimuon resonances
+  # "GoodPFIsoDimuonVertexResonancesNotFromALP",  # Dimuon resonances
+  # "BestPFIsoDimuonVertexNonresonancesNotFromALP",  # Non-resonant Dimuons
+  # "GoodPFIsoDimuonVertexNonresonancesNotFromALP",  # Non-resonant Dimuons
```

Comment:
I guess there's a good reason you commented them out. Could you add some comments on top of this variable to explain what happens when we create an empty list (as in, "if this list is empty, no histograms for X and Y will be plotted")?